components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsSessionCount/src/IsAnalyticsSessionCount.jsx

```diff
@@ -61,50 +62,49 @@ class IsAnalyticsSessionCount extends Widget {
         this.handleUserSelection = this.handleUserSelection.bind(this);
         this.assembleQuery = this.assembleQuery.bind(this);
     }
-    handleResize() {
-        this.setState({width: this.props.glContainer.width, height: this.props.glContainer.height});
 
+    handleResize() {
+        this.setState({ width: this.props.glContainer.width, height: this.props.glContainer.height });
     }
+
     componentDidMount() {
         super.subscribe(this.handleUserSelection);
         super.getWidgetConfiguration(this.props.widgetID)
             .then((message) => {
                 this.setState({
-                    providerConfig: message.data.configs.providerConfig
+                    providerConfig: message.data.configs.providerConfig,
                 });
-            })
+            });
     }
 
     componentWillUnmount() {
         super.getWidgetChannelManager().unsubscribeWidget(this.props.id);
     }
 
     handleDataReceived(message) {
-        let {metadata, data} = message;
         message.data = message.data.reverse();
         this.setState({
             metadata: message.metadata,
-            data: message.data
+            data: message.data,
         });
-
     }
 
     handleUserSelection(message) {
         this.setState({
-          fromDate: message.from,
-          toDate: message.to,
+            fromDate: message.from,
+            toDate: message.to,
         }, this.assembleQuery);
     }
 
     assembleQuery() {
         super.getWidgetChannelManager().unsubscribeWidget(this.props.id);
-        let dataProviderConfigs = _.cloneDeep(this.state.providerConfig);
-        let query = dataProviderConfigs.configs.config.queryData.query;
-        for(let i=0; i<5; i++) {
-        query = query
-            .replace("{{from}}", this.state.fromDate)
-            .replace("{{to}}", this.state.toDate)
-            .replace("{{now}}", new Date().getTime());
+        const dataProviderConfigs = _.cloneDeep(this.state.providerConfig);
+        let { query } = dataProviderConfigs.configs.config.queryData;
+        for (let i = 0; i < 5; i++) {
```

Comment:
Why `5`?

DOn't use magic numbers: https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad